docs/getting-started/endgame-sensor-FDA.asciidoc

```diff
@@ -63,20 +83,14 @@ When you receive the prompt to approve loading the system extension:
 --
 image::images/fda/sec-privacy-pane.png[]
 --
-. On the Security and Privacy pane, select the *Privacy* tab.
-
 +
-. From the left pane, select *Full Disk Access*.
+. Click **Allow** to allow the Endgame system extension to load.
 +
 --
 image::images/fda/select-fda.png[Select Full Disk Access]
```

Comment:
Replace this image with 
`image::images/fda/allow-kernel-ext.png[Allow kernel extension]`

Reply:
Good catch - thanks! Resolved in [2d1674d](https://github.com/elastic/security-docs/pull/3565/commits/2d1674d6f2d1ff9def4ba398af63968adb1ab3c6).